jetty-core/jetty-rewrite/src/main/java/org/eclipse/jetty/rewrite/handler/Rule.java

```diff
@@ -60,61 +60,61 @@ public String toString()
     }
 
     /**
-     * <p>A {@link Request.Wrapper} that is also a {@link org.eclipse.jetty.server.Request.Processor},
+     * <p>A {@link Request.Wrapper} that is also a {@link Handler},
      * used to chain a sequence of {@link Rule}s together.
-     * The rule processor is initialized with the initial request, then it is
+     * The rule handler is initialized with the initial request, then it is
      * passed to a chain of rules before the child {@code Handler} is
-     * passed in {@link #setProcessor(Processor)}. Finally, the response
-     * and callback are provided in a call to {@link #process(Request, Response, Callback)},
-     * which calls the {@link #process(Response, Callback)}.</p>
+     * passed in {@link #setHandler(Handler)}. Finally, the response
+     * and callback are provided in a call to {@link #handle(Request, Response, Callback)},
+     * which calls the {@link #handle(Response, Callback)}.</p>
      */
-    public static class Processor extends Request.Wrapper implements Request.Processor
+    public static class Handler extends Request.Wrapper implements Request.Handler
     {
-        private volatile Processor _processor;
+        private volatile Handler _handler;
 
-        public Processor(Request request)
+        public Handler(Request request)
         {
             super(request);
         }
 
         @Override
-        public boolean process(Request request, Response response, Callback callback) throws Exception
+        public boolean handle(Request request, Response response, Callback callback) throws Exception
         {
-            return process(response, callback);
+            return handle(response, callback);
         }
 
         /**
-         * <p>Processes this wrapped request together with the passed response and
-         * callback, using the processor set in {@link #setProcessor(Processor)}.
+         * <p>Handles this wrapped request together with the passed response and
+         * callback, using the handler set in {@link #setHandler(Handler)}.
          * This method should be extended if additional processing of the wrapped
          * request is required.</p>
          * @param response The response
          * @param callback The callback
          * @throws Exception If there is a problem processing
-         * @see #setProcessor(Processor)
```

Comment:
Change the line above to:
`        * @throws Exception If there is a problem handling`